pkg/cloud/cloud.go

```diff
@@ -110,6 +110,8 @@ const (
 	KubernetesTagKeyPrefix = "kubernetes.io"
 	// AWSTagKeyPrefix is the prefix of the key value that is reserved for AWS.
 	AWSTagKeyPrefix = "aws:"
+	//AwsEbsDriverTagKey is the tag to identify if a volume/snapshot is managed by ebs csi driver
```

Comment:
As i commented above, could you add some more detail about how we use this tag (to enable users to limit the csi driver's permission.

Otherwise, it is difficult to understand why we tag each volume ~4 different ways, as I detailed above